javascript-view-swapping/main.js

```diff
@@ -0,0 +1,32 @@
+const $tabContainer = document.querySelector('.tab-container');
+const $tabElements = document.querySelectorAll('.tab');
+const $viewElements = document.querySelectorAll('.view');
+
+function tabContainerHandler(event) {
+  for (let i = 0; i < $tabElements.length; i++) {
+    if (event.target.matches('.tab') && event.target === $tabElements[i]) {
```

Comment:
You should be checking if `event.target.matches('.tab')` before the loop. We don't want to loop if the target isn't a tab.